src/features/canvass/types.ts

```diff
@@ -60,11 +60,54 @@ export type ZetkinLocationPostBody = Partial<Omit<ZetkinLocation, 'id'>>;
 
 export type ZetkinLocationPatchBody = Partial<Omit<ZetkinLocation, 'id'>>;
 
+//TODO: use from theme object, when canvass is moved to ZUI
+export const householdColors = [
+  '#F1A8A8',
+  '#F7BC9E',
+  '#FDDF91',
+  '#DFEF95',
+  '#BBF1AD',
+  '#99E9CC',
+  '#93E9EB',
+  '#B3DAEE',
+  '#CAC7F7',
+  '#E5C0F5',
+  '#F1A9C9',
+  '#DC2626',
+  '#EA580C',
+  '#FBBF24',
+  '#C0DE2B',
+  '#77E25B',
+  '#34D399',
+  '#28D4D7',
+  '#0284C7',
+  '#4F46E5',
+  '#AA2DDF',
+  '#DB2777',
+  '#841717',
+  '#8C3507',
+  '#977316',
+  '#73851A',
+  '#478837',
+  '#1F7F5C',
+  '#187F81',
+  '#014F77',
+  '#2F2A89',
+  '#661B86',
+  '#831747',
+  '#6D6D6D',
+  '#000000',
+] as const;
+export type HouseholdColor = typeof householdColors[number] | 'clear';
+
 export type Zetkin2Household = {
+  color: HouseholdColor;
   id: number;
   level: number;
   location_id: number;
   title: string;
 };
 
+export type APIHousehold = Omit<Zetkin2Household, 'color'>;
```

Comment:
The name `APIHousehold` felt a bit unusual to me since we don’t usually follow that pattern (at least as far as I’m aware). Something more explicit like `HouseholdWithoutColor` or `HouseholdRecord` might work better? But feel free to disregard the comment if you prefer the current naming!